Favor/Favor/Sources/Scenes/Auth/Coordinator/AuthCoordinator.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  AuthCoordinator.swift
+//  Favor
+//
+//  Created by 이창준 on 2023/01/09.
+//
+
+import OSLog
+import PhotosUI
+import UIKit
+
+import ReactorKit
+
+final class AuthCoordinator: BaseCoordinator {
+  
+  // MARK: - Properties
+  
+  let pickerManager = PHPickerManager()
+  
+  // MARK: - Initializer
+  
+  // MARK: - Functions
+  
+  override func start() {
+    self.showSelectSignInFlow()
+  }
+  
+}
+
+extension AuthCoordinator {
+  
+  /// 로그인 방법을 선택하는 View입니다.
+  func showSelectSignInFlow() {
+    let selectSignInVC = SelectSignInViewController()
+    selectSignInVC.reactor = SelectSignInReactor(coordinator: self)
+    self.navigationController.pushViewController(selectSignInVC, animated: true)
+  }
```

Comment:
`Coordinator`에서 대표적으로 화면전환이 이루어지는건
따로 메서드로 빼지 않아도 될 것 같아 ! 
이런 경우는 조금 보기 헷갈려서 ㅠㅠ..

```suggestion
  override func start() {
    let selectSignInVC = SelectSignInViewController()
    selectSignInVC.reactor = SelectSignInReactor(coordinator: self)
    self.navigationController.pushViewController(selectSignInVC, animated: true)  
    }
}
```

Reply:
오케!